streams/src/test/java/org/apache/kafka/streams/processor/internals/TaskManagerTest.java

```diff
@@ -2779,6 +2779,62 @@ public void suspend() {
         assertThat(task01.state(), is(Task.State.SUSPENDED));
     }
 
+    @Test
+    public void shouldConvertActiveTaskToStandbyTask() {
+        final StreamTask activeTask = mock(StreamTask.class);
+        expect(activeTask.id()).andReturn(taskId00).anyTimes();
+        expect(activeTask.inputPartitions()).andReturn(taskId00Partitions).anyTimes();
```

Comment:
nit:  use `andStubReturn` instead of `andReturn().anyTimes`. No need to change this now, don't want to block the fix, just fyi for future PRs

Reply:
Will fix right away -- Jenkins failed anyway and we need to rerun it.